.env.example

```diff
@@ -5,6 +5,8 @@ ETH_TESTNET_TYPE="pow"
 SP1_PROVER=mock
 # Private key with the permission to use the sp1 network prover (not used if you use SP1_PROVER=mock)
 NETWORK_PRIVATE_KEY="PRIVATE_KEY"
+# Wheter to use a private SP1 cluster (true|false)
+NETWORK_PRIVATE_CLUSTER=false
```

Comment:
We might wanna consider renaming this to something like `E2E_PRIVATE_CLUSTER` so that it is clear that this doesn't affect the relayer or operator normally?

Reply:
Yeah, I guess that might make sense. I'd rather just get this merged right now tho xD

Reply:
Ok, in a future ci PR then